components/dropdown/Dropdown.styled.tsx

```diff
@@ -0,0 +1,121 @@
+/** @jsxImportSource @emotion/react */
+import styled from "@emotion/styled";
+import { DropdownVariant } from "./Dropdown";
+import { QUERIES } from "../../utils";
+
+interface IDropdownStyledProps {
+  isOpen?: boolean;
+  isHighlighted?: boolean;
+  isRHF?: boolean;
+  variant: DropdownVariant;
+}
+
+interface IDropdownContainer {
+  variant: DropdownVariant;
+}
+export const DropdownContainer = styled.div<IDropdownContainer>`
+  width: ${(props) =>
```

Comment:
Nothing to change, but a cool pattern for when there are cases like this is to define a CSS variable and then change that, as it makes declarations more clean, for example:
```css
--containerWidth: ${props => ... ? 125px :  250px}
width: var(--containerWidth); 

```